symphony-app-authentication/pom.xml

```diff
@@ -0,0 +1,153 @@
+        <artifactId>symphonyoss</artifactId>
+        <version>9</version>
+    </parent>
+
+    <groupId>org.symphonyoss.symphony.apps</groupId>
+    <artifactId>symphony-app-authentication</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>Symphony third-party app authentication</name>
+    <description>Components for third-party apps authentication</description>
+    <url>https://github.com/symphonyoss/GroupID/symphony-app-authentication</url>
+
+    <modules>
+        <module>symphony-app-authentication-filter</module>
+    </modules>
+
+    <inceptionYear>2018</inceptionYear>
+    <organization>
+        <name>Symphony</name>
+        <url>https://symphony.com/</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <repositories>
+        <repository>
+            <id>oss-sonatype</id>
+            <name>oss-sonatype</name>
+            <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <developers>
+        <developer>
+            <id>rsanchez</id>
+            <name>Robson Vinicius Vieira Sanchez</name>
+            <email>rsanchez@symphony.com</email>
+            <roles>
+                <role>Senior Software Engineer</role>
+            </roles>
+        </developer>
+    </developers>
+
+    <contributors>
+        <contributor>
+            <name>Adrian Zarifis</name>
+            <email>adrian.zarifis@symphony.com</email>
+            <roles>
+                <role>QA Engineer</role>
+            </roles>
+        </contributor>
+        <contributor>
+            <name>Daniel Nathanson</name>
+            <email>daniel.nathanson@symphony.com</email>
+            <roles>
+                <role>Senior Director</role>
+            </roles>
+        </contributor>
+        <contributor>
+            <name>Vincent Gurle</name>
+            <email>vincent@symphony.com</email>
+            <roles>
+                <role>Product Manager</role>
+            </roles>
+        </contributor>
+    </contributors>
+
+    <issueManagement>
+        <system>JIRA</system>
+        <url>https://perzoinc.atlassian.net/projects/APP</url>
+    </issueManagement>
+
+    <ciManagement>
+        <system>jenkins</system>
+        <url>https://jenkins2.symphony.com</url>
+    </ciManagement>
+
+    <scm>
+        <connection>scm:git:git@github.com:symphonyoss/GroupID.git</connection>
+        <developerConnection>scm:git:git@github.com:symphonyoss/GroupID.git</developerConnection>
+        <url>https://github.com/symphonyoss/GroupID</url>
+    </scm>
+
+    <properties>
+        <!-- JDK -->
+        <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
Is it possible use Java 1.8?

Reply:
No, it isn't in this moment. This library will be used by Integration Bridge that requires Java 1.7